types/src/front/lib/assistant.ts

```diff
@@ -15,7 +15,11 @@ export const MODEL_PROVIDER_IDS = [
 ] as const;
 export type ModelProviderIdType = (typeof MODEL_PROVIDER_IDS)[number];
 
-export const EMBEDDING_PROVIDER_IDS = ["openai", "mistral"] as const;
+export const DEFAULT_EMBEDDING_PROVIDER_ID = "openai" as const;
```

Comment:
is `as const` really needed here ?

Reply:
indeed, don't think it's needed -> I added a commit to remove!